incubator-tools/bank_statement_post_processing_tool/bank_statement_post_processing_tool.ipynb

```diff
@@ -0,0 +1,2834 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2478.        )
</pre><p>This whole code section seems way too long and difficult to read. A lot of this could/should be simplified. But that might be too much to do for this point.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>